creator/settings/production.py

```diff
@@ -20,12 +20,25 @@
 from sentry_sdk.integrations.redis import RedisIntegration
 from graphql import GraphQLError
 
+
+def traces_sampler(sampling_context):
+    """ Filter out unwanted transactions """
+    if (
+        sampling_context.get("wsgi_environ").get("PATH_INFO")
+        == "/health_check"
+    ):
+        return 0
+
+    return 0.1
```

Comment:
I assume you want to return your default sample rate. So maybe make a constant var for this at the module level and use it here and in `init`